### DISCUSSION TITLE
Is there any difference between "2 times in 30 days" and "1 time in 15 days"?

### DISCUSSION THREAD

**Discussion by DanDoes** [iSoron/uhabits#792] [Help & FAQ]
Is there any difference between...
"Repeat 2 times in 30 days" 
...and...
"Repeat 1 times in 15 days"
?

**Comment by hiqua** [comment]
Let's compare 1/2 vs 2/4.
A. 1/2
B. 2/4

The way I understand it, if you have these checkmarks (C where you checked, O if you didn't):
O O C C O O C C

Then in A's case you some missed days (you were supposed to tick on this day), while in B's case the skipped days are still within the margin.

I've switched some habit from weekly to monthly goals to allow for longer breaks for instance (let's say you want to do something 20 times per month but you don't want to be forced to do it on a week-long vacation).

What's still troubling to me is that sometimes you have a missed day today, but if you tick the next few days it will appear to be just a skipped (not missed) day. I think it's counter-intuitive that the result of today (whether the habit was skipped because we could afford it, or missed because we failed) depends on future days.
